src/models/Popup.js

```diff
@@ -452,29 +520,43 @@ export default class Popup extends Base {
     // returns the parent element with the specified class, or the original element - null if not found
     const btn = traverseDOMPath(event.target, 'cc-btn') || event.target
     
-    if (btn.classList.contains( 'cc-btn' ) && btn.classList.contains( 'cc-save' )){
+    if (btn.classList.contains( 'cc-btn' ) && btn.classList.contains( 'cc-customize' )){
+      this.showCustomizeSettingsContent();
+
+      return
+    }
+
+    if (btn.classList.contains('cc-btn') && btn.classList.contains('cc-save')) {
       this.setStatuses()
       this.close(true)
+
       return
     }
-    if (btn.classList.contains( 'cc-btn' )) {
+
+    if (btn.classList.contains('cc-btn')) {
       const matches = btn.className.match(
-        new RegExp('\\bcc-(' + statuses.map( str => str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&') ).join('|') + ')\\b')
```

Comment:
Please extract/use `escapeRegExp` function so the intent is clear in this case.

Reply:
this is original code and we don't use it so let just leave it as it is.